public/electron.js

```diff
@@ -82,15 +83,31 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  // open devtools if DEV_TOOLS is set to 1
+  if (parseInt(process.env.DEV_TOOLS)) {
+    mainWindow.webContents.openDevTools()
+  }
+
   mainWindow.on('closed', () => {
     mainWindow = null
   })
 }
 
 app.on('ready', async () => {
 
-  globalShortcut.register('CommandOrControl+Q', () => app.quit())
+  globalShortcut.register('CommandOrControl+Q', () => {
+
+    // prompt user before closing the application if there are active jobs
+    if (Object.keys(queues['queues']['video transcoding']['childPool'].retained).length != 0) {
+      const dialogOptions = {type: 'info', buttons: ['Cancel', 'Quit'], message: 'There are active jobs running, quit anyway?'}
+      dialog.showMessageBox(dialogOptions, i => {
+        console.log("user has chosen: " + i)
+        if (i == 1) { app.quit() }
```

Comment:
Let's use strict equality `===`. And let's break over multiple lines for readability and consistency.